src/components/app/list-new/AppList.tsx

```diff
@@ -492,7 +496,11 @@ export default function AppList({ isSuperAdmin, appListCount, isArgoInstalled }:
         let queryStr = queryString.stringify(query)
         let _currentTab = selectedAppTab || currentTab
         let url = `${
-            _currentTab == AppListConstants.AppTabs.DEVTRON_APPS ? buildDevtronAppListUrl() : currentTab == AppListConstants.AppTabs.ARGO_APPS  ? buildArgoAppListUrl() : buildHelmAppListUrl()
+            _currentTab == AppListConstants.AppTabs.DEVTRON_APPS
```

Comment:
extract this into some common function to reduce duplicacy. also simplify it with if else to enhance readability